src/app/BufferedReadCallback.h

```diff
@@ -56,7 +56,7 @@ class BufferedReadCallback : public ReadClient::Callback
      *  2. The path provided in aPath is similar to what is buffered but we've hit the end of the report.
      *
      */
-    CHIP_ERROR DispatchBufferedData(const ConcreteAttributePath & aPath, const StatusIB & aStatus, bool aEndOfReport = false);
+    CHIP_ERROR DispatchBufferedData(const ConcreteDataAttributePath & aPath, const StatusIB & aStatus, bool aEndOfReport = false);
```

Comment:
Why this change?

Reply:
ConcreteDataAttributePath has DataVersion inside, ConcreteAttributePath don't have dataVersion. thanks